CHANGES.md

```diff
@@ -73,6 +73,52 @@ To be released.
      -  Added `Transaction<T>.EvaluateActions()` method.
      -  Added `Block<T>.EvaluateActions()` generator method.
 
+ -  The built-in subtype polymorphism of `IAction` and `Transaction<T>` was
+    moved to a separated new `PolymorphicAction<T>` abstract class.
+    Polymorphic actions now should be wrapped by `PolymorphicAction<T>`.
+    For example, the following code:
+
+    ~~~~ csharp
+    public abstract class BaseAction : IAction { ... }
```

Comment:
Can we omit `Base` prefix in docs?

Reply:
It was an intention to emphasize the presence of an `abstract class` as an elaborate example.  Do you prefer simply `Action` over `BaseAction` here?

Reply:
I thought it would be appropriate to show as we were following the [Framework guidelines](https://stackoverflow.com/questions/429470/naming-conventions-for-abstract-classes/429494#429494). but it would be fine if it was intended.

If you intend to make it more clear, it may seem okay to attach an `Abstract` prefix.

Reply:
Applied: 7a6ebc034e102fd2f91ec9e83d4e6f0041a744a7.